addons/website/models/website_menu.py

```diff
@@ -26,6 +27,9 @@ def _default_sequence(self):
     child_id = fields.One2many('website.menu', 'parent_id', string='Child Menus')
     parent_path = fields.Char(index=True)
     is_visible = fields.Boolean(compute='_compute_visible', string='Is Visible')
+    is_mega_menu = fields.Boolean(string='Is Mega Menu')
```

Comment:
There was a warning (in console & runbot) without it. In fact `menu_id.write(menu)` in `save` method tries to find that field. Another way to avoid those warnings is maybe not to write all the `menu` dict but only what is needed

Reply:
Well yes... Why do you even considered adding a new field in the model just because runbot tells you your code seems to write on a field you did not designed :/ Adding a field in a model is something you should not do unless strictly necessary, this is not a random variable you defined, this is column in the database :/